python/lsst/ts/watcher/alarm.py

```diff
@@ -223,6 +230,42 @@ def close(self):
         self._cancel_escalation_timer()
         self._cancel_unmute()
 
+    async def make_log_entry(self, log_server_url):
+        """Post message to narrative log entry in response to alarm"""
+        now = datetime.datetime.now().isoformat()  # use ts_utils repo?
+        # Required? fields in payload:
+        #   message_text, level, user_id, user_agent, is_human
+        msg = f"alarm:{self.name} severity={self.severity}"
```

Comment:
Please add the `self.reason` to the message e.g.;

```
msg = f"alarm:{self.name} severity={self.severity.name} {self.reason}"
```


Also, expand it from `msg` to `message`, maybe even `message_text`, such that you would have:

```
message_text: message_text,
```